.lintstagedrc

```diff
@@ -1,4 +1,4 @@
 {
     "*.{js,json,css,md,ts,tsx}": ["prettier --write", "git add"],
-    "*.{ts,tsx}": ["npm run lint -- --fix", "git add"],
+    "*.{ts,tsx}": ["npm run lint:fix -- --fix", "git add"],
```

Comment:
I think that it will be more useful to move `.lintstagerdc` to each project then it's own config. The inspiration source 👉 https://github.com/sudo-suhas/lint-staged-multi-pkg 🙃 

Because the below command assumes that each package use eslint.
```
    "lint:fix": "eslint 'packages/**/*.{ts,tsx}'"
```

Reply:
> Because the below command assumes that each package use eslint.

Shouldn't that be the case? All packages should be validated the same way using the same config. 🤔

Reply:
In our (developers, owner lib) convention, should be. However, the project shouldn't disallow to use another linter, so the config should be open for extension. 

So the case is, to make root config and extending it in libs, to avoid conflicts when some lib will be stand out with custom config.

of course, each libs can extend its configuration file with a root to avoid duplication like is set `.eslintrc`

https://github.com/Appnroll/appnroll-libs/pull/2/files#diff-74aeb20b276ebc218bdf14cfeeee86cfR2

Reply:
Yup, I agree. With each package having its own configuration we can extend it according to projects needs.